spawn/specification/generator_methods.py

```diff
@@ -74,3 +75,26 @@ def evaluate(self):
         v = self._next_number
         self._next_number += self._step
         return v
+
+
+class ScipyDistribution(Generator):
+    """Generator of values from a statistical distribution in scipy.stats module"""
+    def __init__(self, distribution, random_state=None, **kwargs):
+        """Initialises :class:`ScipyDistribution`
+
+        :param distribution: Name of statistical function that exists in scipy.stats
+        :param kwargs: Arguments to creation of statistical function
+        """
+        try:
+            scipy_stats_module = import_module('scipy.stats')
```

Comment:
Caution advised when using `importlib` functions directly. I would rewrite this to:

```
try:
    import scipy.stats
except ImportError as ex:
    ....
```

Reply:
I changed it as suggested, although that did lead to a strange `AttributeError` which had to be fixed by importing in `conftest.py`.

However, the build is still failing because scipy is not in the environment